sdk/azidentity/azidentity.go

```diff
@@ -54,18 +54,19 @@ func (e *AADAuthenticationFailedError) Error() string {
 	return msg
 }
 
-// AuthenticationFailedError is a struct used to marshal responses when authentication has failed
+// AuthenticationFailedError is returned when the authentication request has failed.
 type AuthenticationFailedError struct {
 	inner error
 	msg   string
 }
 
-// Unwrap method on AuthenticationFailedError provides access to the inner error
+// Unwrap method on AuthenticationFailedError provides access to the inner error authentication
+// error returned from Azure Active Directory
 func (e *AuthenticationFailedError) Unwrap() error {
 	return e.inner
 }
 
-// IsNotRetriable allows retry policy to stop execution in case it receives a AuthenticationFailedError
+// IsNotRetriable allows retry policy to stop execution in case it receives an AuthenticationFailedError
```

Comment:
IsNotRetriable returns true indicating that this is a terminal error.

Reply:
Done